pkg/cli/copy.go

```diff
@@ -56,17 +56,23 @@ func (a *ImageCopy) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if !a.AllTags {
+		if _, err := c.ImageGet(cmd.Context(), args[0]); err == nil {
+			return a.copyLocalToRemote(cmd, c, args, creds)
+		}
```

Comment:
So if the `ImageGet` call errors, what does that mean? Can you add a comment here describing this scenario?

Reply:
I added a comment to the code. If ImageGet returns an error, then either the lookup failed for some reason or the argument just doesn't exist as a local image. Either way we don't really care and can just move on, assuming the user is referring to a remote image.